### ISSUE TITLE
external db password location

### ISSUE THREAD

**Issue by abh** [k8s-land/gitea-chart#7]
At a glance it looks like the chart wants the user to create a secret with the database password when not using the "built-in" MySQL installation.  The documentation / values.yaml file indicates that it can just go there, but the secret doesn't get created.

**Comment by cdrage** [comment]
Yes, you're right. Right now it's hardcoded, when it really shouldn't :( 

I need to update the values.yaml file as well as implement the feature to take it from the secret instead.

**Comment by alexgorbatchev** [comment]
This issue currently makes the chart unusable with external DB unfortunately... unless you manually create the secret first, which is made more difficult by inability to configure secretname/key.

**Comment by cdrage** [comment]
Hey all, I have donated this chart to the folks over at Gitea who have been doing an amazing job maintaining and updating the chart so it is up to date. Please open up any issues here: https://gitea.com/gitea/helm-chart and try theirs own!

Many thanks for all who've contributed.
